### ISSUE TITLE
Frosted Windows Not Blocking Vision

### ISSUE THREAD

**Issue by Kezu02** [space-wizards/space-station-14#21366]
## Description

Places like the Kettle interrogation room or Box lasertag room used to have windows you couldn't see through.

**Screenshots**

![image](https://github.com/space-wizards/space-station-14/assets/143474299/c9238bca-2530-478b-95db-9bbeb71746fe)

**Comment by deltanedas** [comment]
for an interrogation room they should be one way but no idea how that would be implemented

**Comment by IamVelcroboy** [comment]
> for an interrogation room they should be one way but no idea how that would be implemented

We just keep dreaming. Someday....Someday, we'll have these things...

**Comment by mirrorcult** [comment]
non-fulltile entities cant occlude so this isnt possible at the moment
